Tests/Util/UserManipulatorTest.php

```diff
@@ -136,11 +135,10 @@ public function testDeactivateWithValidUsername()
         $this->assertFalse($user->isEnabled());
     }
 
-    /**
-     * @expectedException \InvalidArgumentException
-     */
     public function testDeactivateWithInvalidUsername()
     {
+        $this->expectException(\InvalidArgumentException::class);
```

Comment:
this should better be moved after all the setup part, so that we expect this exception only inside the acting part of the test (which is the reason why PHPUnit deprecated annotations in favor of this API after all).